parser.go

```diff
@@ -278,11 +279,28 @@ type MapType struct {
 	Value *Type `"," @@ ">"`
 }
 
-var parser = participle.MustBuild(&Proto{}, participle.UseLookahead(2), participle.Unquote("String"))
-
 // Parse protobuf.
 func Parse(filename string, r io.Reader) (*Proto, error) {
 	p := &Proto{}
+
+	l := lexer.MustSimple([]lexer.Rule{
+		{"String", `"(\\"|[^"])*"|'(\\'|[^'])*'`, nil},
+		{"Ident", `[a-zA-Z_]([a-zA-Z_0-9])*`, nil},
+		{"Float", `[-+]?((\d*\.\d+)|inf)`, nil},
+		{"Int", `(0[xX][0-9A-Fa-f]+)|([-+]?\d+)`, nil},
+		{"Whitespace", `[ \t\n\r]+`, nil},
+		{"BlockComment", `/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+/`, nil},
+		{"LineComment", `//(.*)[^\n]*\n`, nil},
+		{".", `.`, nil},
```

Comment:
What's this for?

Reply:
That was me being lazy and capturing everything in the lexer as a symbol. Will replace with a list of supported symbols.